### ISSUE TITLE
'.MySportsFeedsEnv' does not appear to be initialized

### ISSUE THREAD

**Issue by znmeb** [MySportsFeeds/mysportsfeeds-r#1]
```
> library("mysportsfeedsR", lib.loc="~/R/x86_64-pc-linux-gnu-library/3.4")
> authenticate_v1_0("znmeb", "notmyrealpassword")
Error in .MySportsFeedsEnv$data$username <- username : 
  object '.MySportsFeedsEnv' not found
> 
```
Should there be some code to create `.MySportsFeedsEnv` if it doesn't exist?

**Comment by bradbarkhouse** [comment]
Yes, and it's in zzz.R, which is supposed to run automatically.  Doesn't seem to be doing that for everyone though.

**Comment by znmeb** [comment]
I'm on Linux, if it matters.

**Comment by bradbarkhouse** [comment]
Not sure, but you can work around it for now by copy/pasting what's in the zzz.R script.

**Comment by znmeb** [comment]
Yeah ... I tried moving zzz.R into the "R" directory and reinstalling - that didn't work. I'm on R 3.4.1

**Comment by aqsmith08** [comment]
I'm getting this error as well. I suspect that because I just signed up the MySportsFeeds username/password isn't active yet. I followed the README with these commands:
`devtools::install_github("MySportsFeeds/mysportsfeeds-r")`
`library(mysportsfeedsR)`
`authenticate_v1_0('USERNAME', 'PASSWORD')`

It resulted in this message:
`
Error in .MySportsFeedsEnv$data$username <- username : 
  object '.MySportsFeedsEnv' not found
`

Any help is greatly appreciated, @bradbarkhouse. Thanks!

**Comment by aqsmith08** [comment]
The workaround of copy/pasting what's in the zzz.R script didn't seem to work either (fyi). I tried to figure it out. :)

**Comment by bradbarkhouse** [comment]
@aqsmith08 you'd only need to copy/paste the inside of the onLoad() handler, lines 2-7.  Essentially all it does is establish a new environment variable named .MySportsFeedsEnv.  Still need to figure out why that zzz.R script doesn't automatically initialize itself.

**Comment by aqsmith08** [comment]
Thanks. That worked. I pasted and ran these commands in my R script:
`.MySportsFeedsEnv <- new.env()`
`.MySportsFeedsEnv$data <- list(
  v1_0_username <- "USERNAME",
  v1_0_password <- "PASSWORD")
`
`authenticate_v1_0('USERNAME', 'PASSWORD')`

This worked.

**Comment by kburnham** [comment]
I was able to fix this by moving the zzz.R file into the R folder and changing the first line to:
`.MySportsFeedsEnv <<- new.env()`
`.MySportsFeedEnv` should now load in the global environment when the package is loaded. 
I submitted a pull request that fixes this. 
In the long run it might be better to do without the .onLoad assignment and have the authenticate function place the username and password in Sys.env(). Would just need to modify msf_getresults to get them from the right place.
